<launch>

    <node pkg="walk_sense" name="walk_forward" type="walk_forward.py" output="screen" />
    <node pkg="hardware_control" name="move_motors_node" type="move_motors_node.py" output="screen" />

</launch>